doc/nrf/releases_and_maturity/migration/migration_guide_2.9.rst

```diff
@@ -28,13 +28,42 @@ Required changes
 
 The following changes are mandatory to make your application work in the same way as in previous releases.
 
+nRF54H20
+========
+
+This section describes the changes specific to the nRF54H20 SoC and DK support in the |NCS|.
+For more information on changes related to samples and applications usage on the nRF54H20 DK, see :ref:`migration_2.9_required_nrf54h`.
+
+DK compatibility
+----------------
+
+.. toggle::
+
+  * The |NCS| v2.9.0 is compatible only with the following versions of the nRF54H20 DK, PCA10175: Engineering C - v0.8.3 and later revisions.
+    The nRF54H20 DK Engineering A and Engineering B (up to v0.8.2) are no longer compatible with the |NCS| v2.9.0.
+
+    Check the version number on your DK's sticker to verify its compatibility with the |NCS|.
+
+Dependencies
+------------
+
+The following required dependencies for the nRF54H20 SoC and DK have been updated.
```

Comment:
Is something missing here?

Reply:
Not really, just preparing the section for later edits